test/SlackMessage.spec.js

```diff
@@ -90,13 +94,21 @@ describe("SlackMessage", () => {
       })
     );
 
+    const postEvents = waitForRequest(
+      server,
+      "POST",
+      "https://slack.com/api/chat.update"
+    );
+
     await message.sendBlock({
       type: "section",
       text: {
         type: "mrkdwn",
         text: ":grey_exclamation:   api starting build",
       },
     });
+
+    await postEvents;
```

Comment:
Awaiting this allows me to produce a test that fails if we call `slackMessage.sendBlock()` that doesn't result in a call to the slack api endpoint called `/api/chat.update`